### PR TITLE
[Translation] Russian translation update

### PR DESCRIPTION
**Pull request by nagayev2002**

### CODE REVIEW

File: base/setup/usetup/lang/ru-RU.h [hunk 1]
```diff
@@ -12,37 +12,37 @@ static MUI_ENTRY ruRULanguagePageEntries[] =
     {
         4,
         3,
-        " ��⠭���� ReactOS " KERNEL_VERSION_STR,
```
Comment (HBelusca) [inline base/setup/usetup/lang/ru-RU.h]: Hi, what has been changed in this file precisely? I fail to see any visual differences...

Comment (binarymaster) [reply base/setup/usetup/lang/ru-RU.h]: Encoding, probably. He used GitHub online source editor.

Comment (HBelusca) [reply base/setup/usetup/lang/ru-RU.h]: If so, that's definitively the wrong way to proceed.

Comment (nagayev) [reply base/setup/usetup/lang/ru-RU.h]: @binarymaster You're right.

Comment (nagayev) [reply base/setup/usetup/lang/ru-RU.h]: @HBelusca Ok, I will consider.

**Comment by nagayev** [comment]
@ThFabba yoops!
I wil try to fix this.
Encoding wrong,I think

File: base/setup/usetup/lang/ru-RU.h [hunk 2]
```diff
@@ -134,61 +134,61 @@ static MUI_ENTRY ruRUIntroPageEntries[] =
...
         12,
-        "and is under heavy development. It is recommended to use it only for",
+        "и в активной разработке. Рекомендуется использовать только для",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         13,
-        "evaluation and testing purposes and not as your daily-usage OS.",
+        "оценики и тестирования,а не как ОС для ежедневного использования.",
```
Comment (Getequ) [inline base/setup/usetup/lang/ru-RU.h]: need space after ","

File: base/setup/usetup/lang/ru-RU.h [hunk 3]
```diff
@@ -134,61 +134,61 @@ static MUI_ENTRY ruRUIntroPageEntries[] =
...
         8,
-        "ReactOS Version Status",
+        "Состояние версии ReactOS",
         TEXT_STYLE_HIGHLIGHT
     },
     {
         6,
         11,
-        "ReactOS is in Alpha stage, meaning it is not feature-complete",
+        "ReactOS находится в Альфа стадии,некоторая функциональность может отсутствовать",
```
Comment (Getequ) [inline base/setup/usetup/lang/ru-RU.h]: space after ,

File: base/applications/cmdutils/mode/lang/ru-RU.rc [hunk 4]
```diff
@@ -67,5 +67,5 @@ BEGIN
     IDS_ERROR_INVALID_PARITY_BITS "ОШИБКА: Неверное значение битов четности %d:\n"
     IDS_ERROR_INVALID_STOP_BITS "ОШИБКА: Неверное значение стоповых битов %d:\n"
     IDS_ERROR_NO_MEMORY "ОШИБКА: Недостаточно памяти.\n"
-    IDS_ERROR_SCREEN_LINES_COL "The screen cannot be set to the number of lines and columns specified.\n"
+    IDS_ERROR_SCREEN_LINES_COL "На экране не может быть задано количество строк и столбцов.\n"
```
Comment (Getequ) [inline base/applications/cmdutils/mode/lang/ru-RU.rc]: добавь "указанное" перед "количество"

File: base/setup/usetup/lang/ru-RU.h [hunk 5]
```diff
@@ -134,61 +134,61 @@ static MUI_ENTRY ruRUIntroPageEntries[] =
     {
         4,
         3,
-        " ReactOS " KERNEL_VERSION_STR " Setup ",
+        " Установка " KERNEL_VERSION_STR " ReactOS ",
```
Comment (Getequ) [inline base/setup/usetup/lang/ru-RU.h]: ReactOS перед " KERNEL_VERSION_STR

File: base/setup/usetup/lang/ru-RU.h [hunk 6]
```diff
@@ -134,61 +134,61 @@ static MUI_ENTRY ruRUIntroPageEntries[] =
...
         16,
-        "to run ReactOS on real hardware.",
+        "запустить ReactOS на реальном устройстве.",
         TEXT_STYLE_NORMAL
     },
     {
         8,
         19,
-        "\x07  Press ENTER to continue ReactOS Setup.",
+        "\x07  Нажмите ENTER чтобы продолжить утсановку ReactOS.",
```
Comment (Getequ) [inline base/setup/usetup/lang/ru-RU.h]: установку

Comment (nagayev) [reply base/setup/usetup/lang/ru-RU.h]: Окей,сегодня исправлю баг с кодировкой и ошибками.

File: base/setup/usetup/lang/ru-RU.h (hunk 2 above)
Comment (r57zone) [reply base/setup/usetup/lang/ru-RU.h]: "оценики" replace to "оценки"

**Comment by HBelusca** [comment]
Hi, is the encoding fixed now? (please never use GitHub editor, but use an editor that conserves encoding, for example Notepad++ on Windows, etc...)

File: base/setup/usetup/lang/ru-RU.h (hunk 2 above)
Comment (cher-nov) [reply base/setup/usetup/lang/ru-RU.h]: и ещё "оценки", а не "оценИки".

**Pull request closed without merging**